No index type in stats result after HNSW_PQ/SQ index created

**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**
```
_____________________________________________________ TestStatsBase.test_get_collection_stats_after_index_created[get_simple_index7] ______________________________________________________

self = <test_collection_stats.TestStatsBase object at 0x7f92b94d1ef0>, connect = <milvus.client.stub.Milvus object at 0x7f92bad47b70>, collection = 'test_jwAmDp6e'
get_simple_index = {'index_type': 'RHNSW_PQ', 'metric_type': 'L2', 'params': {'M': 48, 'PQM': 16, 'efConstruction': 500}}

    def test_get_collection_stats_after_index_created(self, connect, collection, get_simple_index):
        '''
        target: test collection info after index created
        method: create collection, add vectors, create index and call collection_stats
        expected: status ok, index created and shown in segments
        '''
        ids = connect.insert(collection, default_entities)
        connect.flush([collection])
        connect.create_index(collection, default_float_vec_field_name, get_simple_index)
        stats = connect.get_collection_stats(collection)
        logging.getLogger().info(stats)
        assert stats["row_count"] == default_nb
        for file in stats["partitions"][0]["segments"][0]["files"]:
            if file["field"] == default_float_vec_field_name and file["name"] != "_raw":
                assert file["data_size"] > 0
>               assert file["index_type"] == get_simple_index["index_type"]
E               KeyError: 'index_type'

collection/test_collection_stats.py:226: KeyError
------------------------------------------------------------------------------------ Captured log call ------------------------------------------------------------------------------------
[2020-09-25 11:38:42-INFO-root]: {'data_size': 1240653, 'partition_count': 1, 'partitions': [{'data_size': 1240653, 'id': 2273, 'row_count': 1200, 'segment_count': 2, 'segments': [{'data_size': 1111041, 'files': [{'data_size': 12100, 'field': '_id', 'name': '_raw', 'path': '/C_1922/P_2273/S_2205/F_14097'}, {'data_size': 11224, 'field': '_id', 'name': '_blf', 'path': '/C_1922/P_2273/S_2205/F_14102'}, {'data_size': 12100, 'field': 'int64', 'name': '_raw', 'path': '/C_1922/P_2273/S_2205/F_14100'}, {'data_size': 8100, 'field': 'float', 'name': '_raw', 'path': '/C_1922/P_2273/S_2205/F_14098'}, {'data_size': 516100, 'field': 'float_vector', 'name': '_raw', 'path': '/C_1922/P_2273/S_2205/F_14099'}, {'data_size': 400159, 'field': 'float_vector', 'index_type': 'RHNSW_PQ', 'name': '', 'path': '/C_1922/P_2273/S_2205/F_14109'}, {'data_size': 151258, 'field': 'float_vector', 'name': '_compress', 'path': '/C_1922/P_2273/S_2205/F_14110'}], 'id': 2205, 'row_count': 1000}, {'data_size': 129612, 'files': [{'data_size': 5700, 'field': '_id', 'name': '_raw', 'path': '/C_1922/P_2273/S_2206/F_14103'}, {'data_size': 6812, 'field': '_id', 'name': '_blf', 'path': '/C_1922/P_2273/S_2206/F_14108'}, {'data_size': 5700, 'field': 'int64', 'name': '_raw', 'path': '/C_1922/P_2273/S_2206/F_14106'}, {'data_size': 4900, 'field': 'float', 'name': '_raw', 'path': '/C_1922/P_2273/S_2206/F_14104'}, {'data_size': 106500, 'field': 'float_vector', 'name': '_raw', 'path': '/C_1922/P_2273/S_2206/F_14105'}], 'id': 2206, 'row_count': 200}], 'tag': '_default'}], 'row_count': 1200} (test_collection_stats.py:221)

```
**Steps/Code to reproduce behavior**
Follow this [guide](http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) to craft a minimal bug report. This helps us reproduce the issue you're having and resolve the issue more quickly.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Method of installation**
- [x] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Softwars conditions (OS, CPU, GPU, Memory)


- Milvus version (master or released version)


**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
paste-file-content-here
```

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

--- a/tests/milvus_python_test/collection/test_collection_stats.py
+++ b/tests/milvus_python_test/collection/test_collection_stats.py
@@ -221,9 +221,10 @@ class TestStatsBase:
         logging.getLogger().info(stats)
         assert stats["row_count"] == default_nb
         for file in stats["partitions"][0]["segments"][0]["files"]:
-            if file["field"] == default_float_vec_field_name and file["name"] != "_raw":
+            if file["field"] == default_float_vec_field_name and "index_type" in file:
                 assert file["data_size"] > 0
                 assert file["index_type"] == get_simple_index["index_type"]
+                break
 
     def test_get_collection_stats_after_index_created_ip(self, connect, collection, get_simple_index):
         '''

--- a/tests/milvus_python_test/collection/test_collection_stats.py
+++ b/tests/milvus_python_test/collection/test_collection_stats.py
@@ -240,9 +240,10 @@ class TestStatsBase:
         stats = connect.get_collection_stats(collection)
         assert stats["row_count"] == default_nb
         for file in stats["partitions"][0]["segments"][0]["files"]:
-            if file["field"] == default_float_vec_field_name and file["name"] != "_raw":
+            if file["field"] == default_float_vec_field_name and "index_type" in file:
                 assert file["data_size"] > 0
                 assert file["index_type"] == get_simple_index["index_type"]
+                break
 
     def test_get_collection_stats_after_index_created_jac(self, connect, binary_collection, get_jaccard_index):
         '''
@@ -256,9 +257,10 @@ class TestStatsBase:
         stats = connect.get_collection_stats(binary_collection)
         assert stats["row_count"] == default_nb
         for file in stats["partitions"][0]["segments"][0]["files"]:
-            if file["field"] == default_float_vec_field_name and file["name"] != "_raw":
+            if file["field"] == default_float_vec_field_name and "index_type" in file:
                 assert file["data_size"] > 0
                 assert file["index_type"] == get_simple_index["index_type"]
+                break
 
     def test_get_collection_stats_after_create_different_index(self, connect, collection):
         '''
@@ -274,9 +276,10 @@ class TestStatsBase:
             stats = connect.get_collection_stats(collection)
             assert stats["row_count"] == default_nb
             for file in stats["partitions"][0]["segments"][0]["files"]:
-                if file["field"] == default_float_vec_field_name and file["name"] != "_raw":
+                if file["field"] == default_float_vec_field_name and "index_type" in file:
                     assert file["data_size"] > 0
                     assert file["index_type"] == index_type
+                    break
 
     def test_collection_count_multi_collections(self, connect):
         '''
@@ -324,10 +327,12 @@ class TestStatsBase:
             stats = connect.get_collection_stats(collection_list[i])
             if i % 2:
                 for file in stats["partitions"][0]["segments"][0]["files"]:
-                    if file["field"] == default_float_vec_field_name and file["name"] != "_raw":
+                    if file["field"] == default_float_vec_field_name and "index_type" in file:
                         assert file["index_type"] == "IVF_SQ8"
+                        break
             else:
                 for file in stats["partitions"][0]["segments"][0]["files"]:
-                    if file["field"] == default_float_vec_field_name and file["name"] != "_raw":
+                    if file["field"] == default_float_vec_field_name and "index_type" in file:
                         assert file["index_type"] == "IVF_FLAT"
+                        break
             connect.drop_collection(collection_list[i])

